### PR TITLE
Deployment

### PR DESCRIPTION
**Pull request by cadecairos**
Land after #27

### CODE REVIEW

File: scripts/heroku_startup.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+cd app
+
+# Run Django migrations.
+python manage.py migrate
+
+# Start up the system
+gunicorn networkapi.wsgi:application
```
Comment (gideonthomas) [inline scripts/heroku_startup.sh]: blank line at the end

File: app.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "network-api",
+  "description": "API and CMS for The Mozilla Network site",
+  "addons": [
+    "heroku-postgresql"
+  ],
+  "scripts": {
+    "postdeploy": "python app/manage.py migrate"
+  }
```
Comment (gideonthomas) [inline app.json]: can you add the env config as well so that we can enable review apps?

Comment (cadecairos) [inline app.json]: Can you elaborate what you mean by add the env?
Can we not just have review apps inherit their environment variables from staging?

Comment (gideonthomas) [inline app.json]: yeah but to do that you need to specify all the environment vars here according to the heroku docs iirc

**Comment by gideonthomas** [comment]
@cadecairos also add a `runtime.txt` file to specify the python version it should use

**Comment by cadecairos** [comment]
@gideonthomas Added the env vars.

**Pull request merged**